website/docs/user_attributes.md

```diff
@@ -0,0 +1,166 @@
+|          **OIDC scope** | `legal_profile`                                                                               |
+|       **OIDC claim(s)** | `legal_person_identifier`                                                                     |
+| **OIDC claim location** | Userinfo endpoint                                                                             |
+|              **origin** | Provided by user's Identity Provider                                                          |
+|             **changes** | Yes                                                                                           |
+|        **multiplicity** | Single-valued                                                                                 |
+|        **availability** | Always                                                                                        |
+|             **example** | _CA/CA/UniqueId1_                                                                             |
+|               **notes** | -                                                                                             |
+|              **status** | Stable                                                                                        |
+
+### 9. Legal Address
+
+|          attribute name | Legal Name                                                                                      |
+| ----------------------: | :---------------------------------------------------------------------------------------------- |
+|         **description** | The user's legal address                                                                        |
+|   **SAML Attribute(s)** | `http://eidas.europa.eu/attributes/legalperson/LegalAddress` (LegalAddress)                     |
+|          **OIDC scope** | `legal_address`                                                                                 |
+|       **OIDC claim(s)** | `legal_address`                                                                                 |
+| **OIDC claim location** | Userinfo endpoint                                                                               |
+|              **origin** | Provided by user's Identity Provider                                                            |
+|             **changes** | Yes                                                                                             |
+|        **multiplicity** | Single-valued                                                                                   |
+|        **availability** | Always                                                                                          |
+|             **example** | _PGVpZGFzLWxlZ2FsOkZ1bGxDdmFkZHJlc3M+TGVnYWwgYWRkcmVzczwvZWlkYXMtbGVnYWw6RnVsbEN2YWRkcmVzcz4K_  |
+|               **notes** | -                                                                                               |
```

Comment:
notes: Address data is structured as an XML sequence of xsd:string elements. This data is base64 encoded.